specifyweb/specify/urls.py

```diff
@@ -11,6 +11,7 @@
     # replace record
     url(r'^specify/(?P<model_name>\w+)/replace/(?P<new_model_id>\d+)/$', views.record_merge),
     url(r'^specify/merging_status/(?P<merge_id>[0-9a-fA-F-]+)/$', views.merging_status),
+    url(r'^specify/abort_merge/(?P<merge_id>[0-9a-fA-F-]+)/$', views.abort_merge_task),
```

Comment:
why not put all merging APIs into a separate namespace?

`/specify/merge/abort/.../`?
`/specify/merge/status/...//`?
would also avoid the weird `_` in the path (we might be the only ones with `_` in the API URLs)

until we release merging for real it's easier to do such a change.

> REST API resource names
A resource in a REST API is always represented as the plural of an entity that is exposed by the API.
Resource names exposed in a REST API should use all lowercase characters.
Resource names may include hyphens.
Resource names should not include underscores or other punctuation (sole exception is the hyphen).

https://specs.openstack.org/openstack/api-wg/guidelines/naming.html